### PR TITLE
[Backport] add linear strategy to brute force

### LINKED ISSUE #25917: Allow increasing wait time on each failure after the max number of failures is reached
The server will only increase the wait time on multiples of the value set to `Max Login Failures`:

`Wait Increment`  | `Failure Count` | `Max Login Failures` | `Effective Wait Time`
-- | -- | -- | --
30 | 1 | 5 | 6
30 | 2 | 5 | 12
30 | 3 | 5 | 18
30 | 4 | 5 | 24
**30** | **5** | 5 | **30**
30 | 6 | 5 | 36
30 | 7 | 5 | 42
30 | 8 | 5 | 48
30 | 9 | 5 | 54
**30** | **10** | 5 | **60**

While this behavior is valid and expected by some use cases, others might want to increase the wait time linearly on each authentication failure attempt (after the user is no longer locked).

`Wait Increment`  | `Failure Count` | `Max Login Failures` | `Effective Wait Time`
-- | -- | -- | --
30 | 1 | 5 | -90
30 | 2 | 5 | -60
30 | 3 | 5 | -30
30 | 4 | 5 | 0
**30** | 5 | 5 | **30**
**30** | **6** | 5 | **60**
30 | 7 | 5 | 90
30 | 8 | 5 | 120
30 | 9 | 5 | 150
30 | 10 | 5 | 180

A WIP is here https://github.com/keycloak/keycloak/pull/25361.

### PR DESCRIPTION
**Pull request by pedroigor**
closes #25917

### CODE REVIEW

**Review by ahus1 — APPROVED** [review]
Approving based on parent PR review

**Review by mposolda — APPROVED** [review]

**Comment by ahus1** [comment]
@keycloak/ui-maintainers - this is waiting for a review from a UI maintainer

**Review by edewit — APPROVED** [review]

**Pull request merged**
